core.ts

```diff
@@ -13,6 +13,9 @@ import Clipboard from './modules/clipboard';
 import History from './modules/history';
 import Keyboard from './modules/keyboard';
 import Uploader from './modules/uploader';
+import Delta, { Op, OpIterator, AttributeMap } from 'quill-delta';
+
+export { Delta, Op, OpIterator, AttributeMap };
```

Comment:
Export quill-delta so users don't need to install quill-delta on their side. This is useful to prevent multiple versions of quill-delta coexist in a project and `Delta.registerEmbed()` being called on a wrong Delta instance.

An alternative is to get `Delta` from `Quill.import('delta')` but due to [a TypeScript limit](https://github.com/microsoft/TypeScript/issues/36348), users have to use `typeof Delta` instead of `Delta` when referring the type.